src/script/util/DebugUtil.js

```diff
@@ -31,6 +32,9 @@ import {BackendEvent} from '../event/Backend';
 import {StorageSchemata} from '../storage/StorageSchemata';
 import {EventRepository} from '../event/EventRepository';
 
+import {h, init} from 'snabbdom';
```

Comment:
Maybe import `h` and `init` with other names (like `snabbdomInit`?)

Reply:
Good idea for `init`. Will keep `h` as is, it's a convention more than anything else, I think it has value to keep it this way

Reply:
What value would that be? It's hard to figure out what `h` is supposed to mean.

Reply:
It comes from `hyperscript`, it's a, kind of, standard in the vdom dev community
https://github.com/hyperhype/hyperscript

Reply:
The idea, I believe, is to make it short, because you are going to use it (potentially) a lot. Would you prefer a 2 letters word like `el`?

Reply:
I understand that it makes sense when you use it a lot. But since we use it only once, I'd call it `snabbdomHelper`, since `h()` is [documented](https://github.com/snabbdom/snabbdom#inline-example) as a "helper function".

Reply:
What do you think of `createElement` then. At least it explains what it does? (while `snabbdomHelper` doesn't really describe this)

Reply:
Sounds good!